docs/code/spike_times.cpp

```diff
@@ -0,0 +1,61 @@
+// clang++ -std=c++11 -O2 spike_times.cpp -o spike_times
+#include <nix.hpp>
+#include <chrono>
+#include <iostream>
+#include <random>
+
+
+double store_spike_times(nix::DataArray &array, double dt, double spike_prob, size_t chunksize=1000) {
+    auto start = std::chrono::high_resolution_clock::now();
+
+
+    auto end = std::chrono::high_resolution_clock::now();
+    std::chrono::duration<double> elapsed = end - start;
+
+    return elapsed.count();
+}
+
+double store_binary_spikes(nix::DataArray &array, double dt, double spike_prob, size_t chunksize=1000) {
+    auto start = std::chrono::high_resolution_clock::now();
+
+
+    auto end = std::chrono::high_resolution_clock::now();
+    std::chrono::duration<double> elapsed = end - start;
+
+    return elapsed.count();
+}
+
+
+void create_spike_times() {
+    /*
+    std::random_device rd;  //Will be used to obtain a seed for the random number engine
+    std::mt19937 gen(rd()); //Standard mersenne_twister_engine seeded with rd()
+    std::uniform_real_distribution<> dis(0.0, 1.0);
+    for (int n = 0; n < 10; ++n) {
+        // Use dis to transform the random unsigned int generated by gen into a
+        // double in [1, 2). Each call to dis(gen) generates a new random double
+        std::cout << dis(gen) << ' ';
+    }
+    std::cout << '\n';
+    */
+}
```

Comment:
Uncommented on purpose?

Reply:
not yet done, will be used in the future